core/iwasm/interpreter/wasm.h

```diff
@@ -25,6 +25,10 @@ extern "C" {
 #define VALUE_TYPE_V128 0x7B
 #define VALUE_TYPE_FUNCREF 0x70
 #define VALUE_TYPE_EXTERNREF 0x6F
+#define VALUE_TYPE_STRINGREF 0x64
+#define VALUE_TYPE_STRINGVIEWWTF8 0x63
+#define VALUE_TYPE_STRINGVIEWWTF16 0x62
+#define VALUE_TYPE_STRINGVIEWITER 0x61
```

Comment:
The four VALUE_TYPE_STRINGREF types are not used internally, and the similar enum values have been defined in wasm_export.h, how about removing them here? Just keep REF_TYPE_STRINGREFxxx below.

Reply:
Great idea! 
By the way, the similar enum values have been defined in `gc_export.h`, not `wasm_export.h` :)